### PR TITLE
feat: create starting points for intro pages

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Discussed in platform meeting - step one in overhauling these to be proper landing pages.

Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Review by moT01 — APPROVED** [review]
This LGTM if we want it like this. I thought we just wanted to remove the links where the challenges didn't have a name. e.g. all the "step" and "task" links, but keep the ones in the old JS and elsewhere where the challenges have titles. I would probably prefer to just remove them all like this PR does for consistency.

I would replace the `hr` on line 58 with a bigger spacer. Also, I noticed that the footer floats up if there's not enough content to push it to the bottom. That's probably something for another PR.

I would be fine with this as-is - feel free to replace that hr if you think it's appropriate @naomi-lgbt

**Comment by naomi-lgbt** [comment]
> Also, I noticed that the footer floats up if there's not enough content to push it to the bottom. That's probably something for another PR.

That shouldn't be a problem once these have content. Maybe I'll do that all in one PR.

File: client/src/pages/learn/2022/responsive-web-design/learn-css-animation-by-building-a-ferris-wheel/index.md [hunk 1]
```diff
@@ -1,9 +1,32 @@
...
+
+You will learn techniques like using CSS `keyframes` to control how your elements behave.
+
+```css
+@keyframes spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
```
Comment (ilenia-magoni) [inline client/src/pages/learn/2022/responsive-web-design/learn-css-animation-by-building-a-ferris-wheel/index.md:19]: in the project we use only percentages, not `from` and `to`

Comment (naomi-lgbt) [reply client/src/pages/learn/2022/responsive-web-design/learn-css-animation-by-building-a-ferris-wheel/index.md:19]: Yeah there's going to be a lot to improve with the copy for these. I'm just trying to get a basic starting point for now.

**Comment by naomi-lgbt** [comment]
Okay, this should now be in a decent enough state to give us a baseline to start expanding on these as necessary.

**Review by Sembauke — APPROVED** [review]

**Pull request merged**